#
#

Header header
uint8[16] firstLine     # Up to 32 characters, 16 in the first row, 16 in the second row;
uint8[16] secondLine
